src/helpers/getCSSVariable.tsx

```diff
@@ -0,0 +1,7 @@
+export default function getCSSVariable(name: string) {
+    let propertyValue = "";
+    propertyValue = getComputedStyle(document.documentElement).getPropertyValue(
+        name
+    );
```

Comment:
You can make the variable `const` and at the end of line 5 do `|| "";`. So if `getComputedStyle` doesn't return a value, then the fallback is the empty string.